### DISCUSSION TITLE
How to reach this code?

### DISCUSSION THREAD

**Discussion by idamir** [Qalculate/qalculate-qt#63] [Show and tell]
I want to translate this string correctly, but I do not understand how to reach this point to check results.
https://github.com/Qalculate/qalculate-qt/blob/036bd5bb6603567b2de6101c972d6670f2535e7d/src/historyview.cpp#L527

**Comment by hanna-kn** [comment]
The the string is shown in a tooltip when the pointer is over the flag displayed after a currency in the result list.
